src/components/TimeTable/timeTable.js

```diff
@@ -1,7 +1,26 @@
 import React from "react";
 
 export default function TimeTable() {
+
     return (
-        <h1>TimeTable</h1>
+        <div className="table">
+            <button onClick={loadTable()}>Kliknij</button>
+            <div className="content"></div>
+        </div>
+
+
     )
+    function loadTable()  {
+        var xhttp = new XMLHttpRequest();
```

Comment:
No poooprawnie noo, ale zdecydowanie nie lubisz ułątwiać sobie życia :P
Jest sporo sposobów, żeby cąły ten kod zmieścić w 2 linijkach

Reply:
Bo to jeszcze nie ta wersja xD
Bedzie poprawniej w nastepnym gicie xD